gulpfile.js

```diff
@@ -49,7 +49,7 @@ gulp.task('compile_js', function() {
         'only_closure_dependencies': true,
         'output_wrapper': '(function(){%output%})();',
         'js': [
-            './bower_components/closure-library/**.js',
+            './bower_components/closure-library/closure/**.js',
```

Comment:
I think this is good -- but should we also add a line for `third_party` inside closure-library?

https://github.com/google/closure-library/tree/master/third_party/closure/goog

`goog.dom.query` (and others) require this search path.